internal/commandsgen/code.go

```diff
@@ -98,6 +123,36 @@ func (c *codeWriter) importPkg(pkg string) string {
 
 func (c *codeWriter) importCobra() string { return c.importPkg("github.com/spf13/cobra") }
 
+// appendGoSource parses a Go source file, registers its imports, and appends
+// everything after the import block to the output buffer.
+func (c *codeWriter) appendGoSource(src string) error {
```

Comment:
It was either this, string literals or regex; this seems like the best approach to me.